Builds sometimes fail with Could not resolve dependencies for project

Example log:
```
+ ./mvnw install -DskipTests -Dair.check.skip-all=true -Dmaven.javadoc.skip=true -B -q -T C1 -pl '!presto-docs,!presto-server-rpm'

[ERROR] Failed to execute goal on project presto-parser: Could not resolve dependencies for project io.prestosql:presto-parser:jar:302-e.0-SNAPSHOT: Failed to collect dependencies at io.airlift:slice:jar:0.36: Failed to read artifact descriptor for io.airlift:slice:jar:0.36: Could not transfer artifact io.airlift:slice:pom:0.36 from/to starburst_nexus (http://nexus.aws.starburstdata.com:8081/nexus/content/repositories/central): /home/ubuntu/.m2/repository/io/airlift/slice/0.36/slice-0.36.pom.part (No such file or directory) -> [Help 1]
```

This could be related to: https://jira.apache.org/jira/browse/MDEP-518

It is suggested there that Takari's builder fixes the issue. I've seen that we had Takari as Maven extension once, but was removed via: ﻿﻿﻿﻿https://github.com/prestosql/presto/commit/33e41f3b7ed2c2129103d74c33943d8cf4114788

FYI @electrum

--- a/.mvn/extensions.xml
+++ b/.mvn/extensions.xml
@@ -1,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+    <extension>
+        <groupId>io.takari.aether</groupId>
+        <artifactId>takari-concurrent-localrepo</artifactId>
+        <version>0.11.2</version>
+    </extension>
+</extensions>

